### ISSUE TITLE
Unable to set favorite from favorite station as input

### ISSUE THREAD

**Issue by jakolion** [foxriver76/ioBroker.denon#76]
Hi,
I have a Denon AVR-X2000 and trying to start one of the three favorites of my favorite station.
With favorite station I mean the three buttons 1-3 which can be found on the remote and the web view:

![image](https://user-images.githubusercontent.com/31564175/81320072-e206ac80-9090-11ea-829e-08557a9adbda.png)

The currently selected input is stored to these buttons when long pressing the corresponding number on the remote. They are different from the favorites in the favorite list where you can have much more than theses three entries.

After looking at the documentation of this plugin, I thought they are called "quickSelect" and that I have to set them. But setting the zoneMain.quickSelect to 1-5 always does something different.

When pressing the button "2" on the web view the following commands are send to the Denon:
cmd0: PutZone_InputFunction/FAVORITE2
cmd1: aspMainZone_WebUpdateStatus/

Can someone help me? Perhaps there is a command I have to send via "settings.expertCommand"?

**Comment by foxriver76** [comment]
Afaik there are no other telnet commands to control favorites. Maybe its only possible via http request. But sounds like the quickSelect Buttons should do the job.

**Comment by jakolion** [comment]
Thank you for your answer, I wanted to make sure there really is no other way. I will solve it with a http request then.
While looking at http requests: there is no way to get the configured IP address via the states of the plugin, am I correct? This would reduce the configuration effort if the IP address changes for some reason (of course will rarely happen) or if you want to use a second instance of this plugin for another device. What do you think? Should I create a feature request regarding this?

**Comment by foxriver76** [comment]
You get the IP address from system.adapter.denon.0 if your instance is 0 then there is the property `native` and there is the property `ip`. 
Js adapter 
```javascript
const ip = getObject(‘system.adapter.denon.0').native.ip
```

In the title your write `favorite station` is it about radio channels? Then you could also try `info.onlinePresets`.

**Comment by jakolion** [comment]
Hmm, then perhaps it is different or not possible within `node-red`, this is the list of values the UI shows me I could get:

![image](https://user-images.githubusercontent.com/31564175/81385810-4454c100-9114-11ea-8b8d-3e27d50d796a.png)

I tried to get `system.adapter.denon.0.native.ip` but it gives me no data, seems `native` is not known here.

The favorite station can store the following contents:

![image](https://user-images.githubusercontent.com/31564175/81386013-9b5a9600-9114-11ea-8983-8cdf5494f8fb.png)

I saw the `info.onlinePresets` but I would like to be able to set the favorites with the remote. For me the solution with http requests is fine.
